docs/mindsdb-docs/docs/sql/tutorials/diabetes.md

```diff
@@ -0,0 +1,290 @@
+For the database to connect to MindsDB we will use a tunneling program Ngrok to allow the remote database connection without exposing your public IP.
+
+Data will be loaded from a csv file using the import tool.
+[Docker Volumes](https://docs.docker.com/storage/volumes/) are used for persistent storage of data.
+
+To run, open a terminal in the project folder and run `docker-compose up -d` .
+If you do not have the Docker images locally, it will first download them from docker hub and might take.
+
+## Setup the database
+
+After docker has started all the services
+In this section, you will create a Postgres database and a table into which you will then load the dataset.
+
+First, connect to your Postgres instance. There are cli options like [pgcli](https://www.pgcli.com/).
+
+We will use a gui manager; **pgAdmin**
+
+> The required details are specified within the compose file. Remember to change the username & password if you have a different one set up in.
+
+1. Open your browser and enter localhost:5050 in the Navigation bar.
+2. Setup or enter a master password.
+![Access pg4admin](/assets/tutorials/diabetes/pg4admin/Accessing_pg4admin.gif)
+3. Create a new server connection by navigating to the right side of the page and right click on 'Server', select 'Create' and enter the required details.
+
+    3.1 For the purpose of the exercise, under the General Tab the Name entered is MIndsDB and the host name/address is localhost under the Connection tab .
+![Create Server](/assets/tutorials/diabetes/pg4admin/Creating_pgserver.gif)
+
+4. To create a new database, select the dropdown of the Server and navigate to 'Databases', right click on it and select 'Create' and then 'Databases'.Enter the required details.
+
+    4.1 For this exercise, the Database was given the name DIABETES_DATA under the General tab.
+![Create Database](/assets/tutorials/diabetes/pg4admin/Creating_pgdatabase.gif)
+
+5. Select the dropdown of the database created, navigate to 'Schema' and right click on it to select 'Create' and then 'Schema'. Enter the required details.
+
+6. Select the dropdown of the schema created and navigate to 'Tables' to right click on it and select 'Query Tool'. This will allow you to add a table with SQL code.The below code was used to create the table.
+
+```sql
+CREATE TABLE Diabetes (
+    Number_of_times_pregnant INT ,
+    Plasma_glucose_concentration DECIMAL(6,3) ,
+    Diastolic_blood_pressure DECIMAL(6,3) ,
+    Triceps_skin_fold_thickness DECIMAL(6,3) ,
+    Two_Hour_serum_insulin DECIMAL(6,3) ,
+    Body_mass_index DECIMAL(6,3) ,
+    Diabetes_pedigree_function DECIMAL(6,3) ,
+    Age INT ,
+    Class varchar(100)
+);
+```
+![Create table](/assets/tutorials/diabetes/pg4admin/Create_pgtable.gif)
+7. Before you import the data, please delete the first row as it is a table header. To import rows into your table, right click on the table name and select 'Import/Export'.
+
+    7.1 Under the Options tab, ensure the slider is set to 'Import'.
```

Comment:
Please fix this as it looks like a code blog

Reply:
amended*